webrender/src/frame.rs

```diff
@@ -320,6 +320,10 @@ impl Frame {
                                                   background_color,
                                                   self.frame_builder_config);
 
+        if let Some(ref mut old_builder) = self.frame_builder {
```

Comment:
I started with something like this and the reason I ended up changing is that if you add members to FrameBuilder it is easy to forget to handle them in clear() which can lead to some subtle misbehavior. With this approach, if you forget to handle the member in recycle you won't have a bug (just potentially miss an optim, which will show up in profiles if it turns out to have any impact).

If you have a strong preference for doing it the other way I can rearrange the patch.